### DISCUSSION TITLE
Photos from Phone

### DISCUSSION THREAD

**Discussion by rjgrandy** [hay-kot/homebox#182] [Ideas]
Would be nice if I could take and attach a photo from my phone.

**Comment by hay-kot** [comment]
It let's me do it on Android. What have you tried?

**Comment by imakichado** [comment]
I can add a picture via my Android, but it automatically comes in as an attachment instead of a photo. I am trying to have aging parents go around and document important-to-the-family/valuable items in their home. It would be helpful if it would default to picture instead of attachment when added via mobile or when the attachment is ends in jpg, etc..
ALSO, if the attachment/picture box could also be on the initial create item form and not only after you open an existing item, click edit, and then advanced.  Homebox is really perfect in our use case - clean, simple interface and not a lot of overly complicated jargon. Thank you!!!
